src/Form/Form.js

```diff
@@ -451,6 +468,9 @@ class Form {
 
```

Comment:
I noticed that if we click again on Dax the tooltip flashes (this is in production as well) and the icon fades back from grayscale to color (the flashing is not very visible in the gif, but you can easily reproduce). What if we just returned if the tooltip is open and the click is on the icon? Probably before (or even within) checking for `shouldOpenTooltip` ([here](https://github.com/duckduckgo/duckduckgo-autofill/pull/278/files#diff-1475870a2b4ab813018df5a922e51a32464916f7c93265a1edebc91a9150fc18R455))?

![CleanShot 2023-03-20 at 09 25 46](https://user-images.githubusercontent.com/1828326/226285045-987499cd-df95-487d-a1b2-833161f607fd.gif)

Reply:
@GioSensation It's an issue of the `HTMLTooltipUIController.js` removing the tooltip (because you've clicked away from it) and the `Form.js` then re-adding the tooltip (because you're clicking on the Dax icon) -- to prevent this flash, we can update `_pointerDownListener` in `HTMLTooltipUIController.js` to bail if `isEventWithinDax`. I've added this change in the latest commit 👍